### PR TITLE
Allow fine grain override control

### LINKED ISSUE #31: Change the override button with a slide so it can be finetuned in real-time

### PR DESCRIPTION
**Pull request by capitancambio**
Closes #31

### CODE REVIEW

**Comment by monchote** [comment]
@capitancambio You're very conflictive.

File: app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java [hunk 1]
```diff
@@ -143,11 +138,42 @@ public void onClick(View v) {
...
+
+            }
+        });
 
         mSpheroActions = new View[]{calibrateLeftButton, calibrateRightButton, panicButton};
-        mMuseActions = new View[]{noFocusButton, horizonButton};
+        mMuseActions = new View[]{toggleButton, horizonButton};
 
         return view;
+
```
Comment (monchote) [inline app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java]: Remove extra line.

File: app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java [hunk 2]
```diff
@@ -143,11 +138,42 @@ public void onClick(View v) {
                 }
             }
         });
+        ToggleButton toggleButton = (ToggleButton) view.findViewById(R.id.toggle_override);
+        toggleButton.setTextOn(getResources().getString(R.string.off));
+        toggleButton.setTextOn(getResources().getString(R.string.override));
```
Comment (monchote) [inline app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java]: setTextOn twice.

Comment (capitancambio) [inline app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java]: I did something weird in the rebase again \o/

Comment (monchote) [inline app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java]: Also, don't set it programatically but in XML with android:textOn and android:textOff.

File: app/src/main/res/layout/fragment_control.xml [hunk 3]
```diff
@@ -198,13 +198,24 @@
...
 
+        <LinearLayout
+            android:orientation="horizontal"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <ToggleButton
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="Override"
```
Comment (monchote) [inline app/src/main/res/layout/fragment_control.xml]: Remove and replace with android:textOn and android:textOff.

**Comment by monchote** [comment]
@capitancambio Conflicts again.

**Pull request merged**